src/platform/site-wide/announcements/components/SingleSignOnInfoModal.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import Modal from '@department-of-veterans-affairs/formation-react/Modal';
+import { hasSessionSSO } from 'platform/user/profile/utilities';
+
+export default function SingleSignOnInfoModal({ dismiss, isLoggedIn }) {
+  if (!isLoggedIn && !hasSessionSSO()) return null;
+
+  return (
+    <Modal visible onClose={dismiss} id="modal-announcment">
+      <h1 className="vads-u-font-size--h3 vads-u-margin-top--2">
+        Sign in once to access the VA sites you use most
+      </h1>
+      <p>
+        Now when you sign into VA.gov, we'll also sign you into MyHealtheVet ,
+        eBenefits, and other VA sites.
+      </p>
+      <p>
+        <strong>With a single sign on, you can:</strong>
+      </p>
+      <ul>
+        <li>
+          Track and manage your VA benefits and services with just one username
+          and password
+        </li>
+        <li>
+          Access the VA sites you use most without having to sign in each time
+        </li>
+        <li>Sign out of VA.gov and be signed out of other VA sites</li>
+      </ul>
+      <button type="button" onClick={dismiss}>
+        Continue to VA.gov
+      </button>
+      <p>
+        Get answers to{' '}
```

Comment:
@hinzed1127 does `{' '}` just force a space between "to" and the anchor?

Reply:
yup, this is a [weird quirk](https://reactjs.org/docs/jsx-in-depth.html#string-literals-1) of JSX. although looks like this is gonna get removed for now anyway 🤷‍♂